bin/cron_pull_tweets.py

```diff
@@ -67,4 +67,8 @@ def run_task(params, ntweets=3, maxpages=2):
         with open('secrets.txt', 'r') as f:
             params = json.load(f)
 
-    run_task(params)
+    ntweets = 100
```

Comment:
Assuming this is used for debugging rn. Make an env var

Reply:
The env var 'P_CRON_NTWEETS' was created (see lines 71–74). Line 70 serves as a fallback default, although perhaps we could refactor that!